test/e2e/v3/generate_test.go

```diff
@@ -228,10 +228,52 @@ Count int `+"`"+`json:"count,omitempty"`+"`"+`
 	if crdAndWebhookVersion == "v1beta1" {
 		_ = pluginutil.RunCmd("Update dependencies", "go", "mod", "tidy")
 	}
+
+	if restrictived {
+		By("uncomment kustomize files to ensure that pods are restricted")
+		uncommentPodStandards(kbc)
+	}
+}
+
+func uncommentPodStandards(kbc *utils.TestContext) {
+	configManager := filepath.Join(kbc.Dir, "config", "manager", "manager.yaml")
+	managerAuth := filepath.Join(kbc.Dir, "config", "default", "manager_auth_proxy_patch.yaml")
```

Comment:
Nit:
```suggestion
	managerAuth := filepath.Join(configManager, "default", "manager_auth_proxy_patch.yaml")
```

Reply:
We cannot do this change. 
Why?
configManager-> has the full path with the file